src/inverseRobot.js

```diff
@@ -8,6 +8,16 @@
 
 function inverseRobot(robot) {
   // write code here
-}
+  const inverse = {};
+
+  for (const key in robot) {
```

Comment:
It's recommended to use `Object.keys()` or `Object.entries()` to iterate over an object. This way, you don't have to worry about properties that come from the object's prototype.